annotation-processor/src/test/java/org/hibernate/validator/ap/testmodel/annotationparameters/InvalidGroupSequenceParameters.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.ap.testmodel.annotationparameters;
+
+import javax.validation.GroupSequence;
+import javax.validation.constraints.AssertFalse;
+import javax.validation.constraints.AssertTrue;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/**
+ * @author Marko Bekhta
+ */
+public class InvalidGroupSequenceParameters {
+
+	/**
+	 * Case 1: class present in a list of groups
+	 */
+	public static class Case1 {
+		public interface Group1 {
+		}
+
+		public interface Group2 {
+		}
+
+		@GroupSequence(value = Group2.class)
+		public interface Group3 {
+		}
+
+		public class InvalidGroup1 {
+		}
+
+		public class InvalidGroup2 {
+		}
+
+		@GroupSequence(value = { Group1.class, InvalidGroup1.class })
+		public class SomeBean {
+		}
+
+		@GroupSequence(value = InvalidGroup2.class)
+		public class SomeOtherBean {
+		}
+
+		@GroupSequence(value = { Group3.class, YetOtherBean.class })
+		public class YetOtherBean {
+
+		}
+	}
+
+	/**
+	 * Case 2: primitive present in a list of groups
+	 */
+	public static class Case2 {
+		public interface Group1 {
+		}
+
+		public interface Group2 {
+		}
+
+		@GroupSequence(value = { Group1.class, int.class })
+		public class SomeBean {
+		}
+
+		@GroupSequence(value = boolean.class)
+		public class SomeOtherBean {
+		}
+
+		@GroupSequence(value = double.class)
```

Comment:
Same as before :) you are right it's just more to cover different values. Maybe the implementation is just looking for `boolean` :) But again based on the fact that we know that it's determining if the provided class is a primitive or not we can leave just one :)